src/components/docs/Cards/ColorCard/index.js

```diff
@@ -12,13 +12,14 @@ const ColorCard = ({ color, icon, overlay, title, to, type, ...props }) => (
   <BaseContainer 
     className={classnames(styles.root, styles[color], {
       [styles.snippetTab]: type === 'snippetTab',
+      [styles.withOverlay]: type === 'snippetTab' && overlay,
```

Comment:
Basically, at around 1000px the left and right sidebar are added in and the main content section is quite small. So it makes sense to return to the layout of headers being overhead instead of side by side so that there is enough space for the snippet. On mobile we don't have to worry about a hover overlay because hover is not really a mobile thing, but because 1000px is a plausible laptop width, it should be accounted ffor